### ISSUE TITLE
multiple emacs25 issues

### ISSUE THREAD

**Issue by ardumont** [org-trello/org-trello#353]
Hello,

After fixing out some of my org-trello tools, it seems there are issues with the code running with emacs 25.

### M-x org-trello-bug-report

```
org-trello - Please:
- Describe your problem with clarity and conciceness (cf. https://www.gnu.org/software/emacs/manual/html_node/emacs/Understanding-Bug-Reporting.html)
- Explicit your installation choice (melpa, marmalade, el-get, tarball, git clone...).
- Activate ‘’trace‘’ in logs for more thorough output in *Message* buffer: (custom-set-variables ’(orgtrello-log-level orgtrello-log-trace)).
- A scrambled sample (of the user’s and board’s ids) of your org-trello buffer with problems.
- Report the following message trace inside your issue.

System information:
- system-type: gnu/linux
- locale-coding-system: utf-8-unix
- emacs-version: GNU Emacs 25.1.1 (x86_64-pc-linux-gnu, GTK+ Version 3.22.5)
 of 2017-01-01, modified by Debian
- org version: 9.0.5
- org-trello version: 0.7.9
- org-trello path: /home/tony/repo/public/org-trello/org-trello.el
- request-backend: curl
- kill-whole-line: nil
```

### Expected behavior

I expect tests pass so that i can ship a new version.

Under an emacs25:
```
make test
```

### Actual behavior

Tests fail

### Steps to reproduce the behavior

```
make test
```

**Comment by ardumont** [comment]
TL;DR

Fortunately, nothing too drastic seems to be the source of problems:
- order in associated list in json-read-from-string has been inverted (in some lower layer, it is used). Not that the order is important.
- the quote ' is not ' as string output so breaking the string equality tests
- some mock tests broke
